### PR TITLE
adding Placemement Binding to glossary 

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-3393

### CODE REVIEW

**Comment by dockerymick** [comment]
@dhaiducek or @JustinKuli can you all help with confirming the definition of the Placement binding. I plan to update the Placement binding API in a separate PR.

File: about/glossary_terms.adoc [hunk 1]
```diff
@@ -95,10 +95,10 @@ Specialized controller on the Klusterlet that provides additional management cap
 
 Created and imported clusters are managed by the klusterlet agent and its add-ons, which initiates a connection to the {product-title} hub cluster.
 
-[#p-term-policy]
-=== Placement policy
+[#p-term-placement-binding]
+=== Placement binding
 
-A policy that defines where the application components should be deployed and how many replicas there should be.
+A policy that binds the placement to a placement rule.
```
Comment (JustinKuli) [inline about/glossary_terms.adoc]:
```suggestion
A resource that binds a placement or placement rule to a policy.
```

I wonder if the original entry was meant to talk about this kind of placement: https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#placement-overview

Comment (dockerymick) [reply about/glossary_terms.adoc]: @JustinKuli ahhhh, good call out! I agree with you. Let me revert my change and add Placement Binding as a separate entry. Idk why I thought placement was suppose to be removed. And i missed my proposed update in the issue template, yikes. Thanks again Justin!

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

**Review by JustinKuli — APPROVED** [review]
Looks good to me

File: apis/placement.json.adoc [hunk 2]
```diff
@@ -1,14 +1,14 @@
-[#placement-binding-api]
-= PlacementBinding API (v1beta1)
+[#placement-api]
+= Placement API (v1beta1)
```
Comment (dockerymick) [inline apis/placement.json.adoc:2]: Previous PR: https://github.com/stolostron/rhacm-docs/pull/4832/files#diff-e1c35eeb8132d16625dcfd6060035447704011bd31442b8a15530fdac26e4d61R1

**Comment by dockerymick** [comment]
@cadawson @JustinKuli thanks for the reviews and approval!

**Pull request merged**